src/MgvRoot.sol

```diff
@@ -138,12 +138,12 @@ contract MgvRoot is HasMgvEvents {
   function configInfo(OLKey memory olKey)
     external
     view
-    returns (MgvStructs.GlobalUnpacked memory global, MgvStructs.LocalUnpacked memory local)
+    returns (MgvStructs.GlobalUnpacked memory _global, MgvStructs.LocalUnpacked memory _local)
```

Comment:
This looks weird - why not avoid the `_` ?

Reply:
We have _global in other function signatures as well. My editor doesn't like just "global" and I thought that was why it was introduced elsewhere, so I aligned it. It is not super important whether we go with or without _, but I think we should be consistent.

Reply:
Agree on the consistency. Unless there's a reason to have the `_` I think we should avoid it.

Reply:
global is a reserved keyword. Agreed with @adhusson to keep _.